### ISSUE TITLE
Error while trying to run the debugger after updating the react-native-tools

### ISSUE THREAD

**Issue by preetb123** [Microsoft/vscode-react-native#305]
[Error] "Could not debug. An error ocurred while handling message: GET_PACKAGER_PORT"

**Comment by MSLaguana** [comment]
Are you running on windows or mac? Did you restart vscode after updating the extension?

**Comment by preetb123** [comment]
OS X EI Capitan v.10.11.3 
VSCode: v. 1.5.3. I also use insiders build but not working their as well.
vscode-react-native: 0.2.0

**Comment by mbraude** [comment]
Hi @preetb123, thanks for reporting this! Is there any more information you can give us? We need to be able to repro this issue before we can do much more to help.

**Comment by preetb123** [comment]
Please let me know what specific information you need to repro this.

**Comment by MSLaguana** [comment]
This can happen when our extension has not properly activated. Have you opened a folder containing a package.json and does `react-native -v` run in that folder return a react-native version, or complain of not being in a react-native project directory?

What happens if you hit `cmd-shift-p` and try to `>React Native: Start Packager`?

**Comment by mbraude** [comment]
@preetb123 is this still an issue for you?

**Comment by preetb123** [comment]
@mbraude this is working in the latest release of code. But for code-insiders, after installing and running app on device, following error occurs
```
[vscode-react-native] Finished executing: adb -s emulator-5554 shell am broadcast -a "com.hellorn.RELOAD_APP_ACTION" --ez jsproxy true
node --debug-brk=48593 --nolazy launchReactNative.js 9090 
Debugger listening on [::]:48593

[vscode-react-native] Starting debugger app worker.
[vscode-react-native] Established a connection with the Proxy (Packager) to the React Native application
[vscode-react-native] Disconnected from the Proxy (Packager) to the React Native application. Retrying reconnection soon...
[vscode-react-native] [Error : 0706]  Cannot debug application: Another debugger is already connected to packager. Please close it before trying to debug with VSCode.
```

@MSLaguana `cmd+shift+p -> React Native: start packager` starts the packager now and works fine, didn't try it that time. I used to use `npm start` from terminal

**Comment by vladimir-kotikov** [comment]
> `[Error : 0706]  Cannot debug application: Another debugger is already connected to packager. Please close it before trying to debug with VSCode.`

@preetb123, this sounds like you have a debugger tab opened in Chrome. Before starting debug session please ensure that previous debugging sessions have have been closed.

**Comment by andrekovac** [comment]
@vladimir-kotikov Everytime I close the previous debugger sessions or even quit Chrome entirely, Chrome opens up again and VSCode does not act as the debugger.

**Comment by SounD120** [comment]
@andrekovac If I understood you correctly - you need to launch packager through the VS Code commands (`React Native: Start Packager` command palette command or any launch scenarios (`Debug Android`, `Debug iOS`, `Debug in Exponent`)). VS Code will launch the packager with some special environment variables keys that will not allow to launch Chrome for debugging, keep in mind that the external packager terminals should be closed, otherwise the 8081 port will be busy, and VS Code will be unable to launch the packager.

**Comment by andrekovac** [comment]
@SounD120 No I don't even have to manually start the packager inside VSCode via `React Native: Start Packager`. That is done automatically by just hitting the green "Start" button in the debug menu.

Another thing I however have to do to get it working is to manually build the app once via an external terminal, then stop the external packager and only then start debugging via VSCode. Somehow VSCode can't kick things of unless I did it once before. Perhaps it's just that VSCode can't open the Simulator, but well, the workaround works for me for now.

**Comment by SounD120** [comment]
@andrekovac Could you please create a new issue and describe your case using our [issue template](https://github.com/microsoft/vscode-react-native/blob/master/.github/ISSUE_TEMPLATE.md). Thank you.
